cdap-cli/src/main/java/io/cdap/cdap/cli/command/app/DeleteAppCommand.java

```diff
@@ -46,8 +45,6 @@ public DeleteAppCommand(ApplicationClient appClient, CLIConfig cliConfig) {
   @Override
   public void perform(Arguments arguments, PrintStream output) throws Exception {
     ApplicationId appId = parseApplicationId(arguments);
-    ApplicationDetail appDetail = appClient.get(appId);
```

Comment:
the command allows somebody to specify a version. See below in getPattern() and getDescription(). Those parts needs to be changed such that version is not longer supported.

Reply:
Yup, removing those functions in the next rev